app/helpers/report_helper.rb

```diff
@@ -33,7 +33,8 @@ def resolutions_count
   end
 
   def avg_first_response_time
-    grouped_reporting_events = (get_grouped_values scope.reporting_events.where(name: 'first_response', account_id: account.id))
+    grouped_reporting_events = (get_grouped_values scope.reporting_events.where(name: 'first_response',
```

Comment:
Do we need this change since we would not record the bot responses in the first_response metric anymore? If this is done to support the previous data. Let's update the previous data using a migration.

Reply:
Yeah, this change becomes redundant over time.

> Let's update the previous data using a migration.
Should I drop the old rows?

Reply:
@pranavrajs we can add a migration to drop the old rows with incorrect data like this

```ruby
class DropReportingEventsWithIncorrectFirstResponses < ActiveRecord::Migration[6.1]
  def change
    Reporting::Event.where(name: 'first_response').where(user_id: nil).delete_all
  end
end
```

Does this work?